README.md

```diff
@@ -94,72 +166,117 @@ JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid
 
-#### Step 7: Generate templates for the metadata
+Each pipeline has a directory in `pipelines` by the name of the `data_indicator` (from the input data). It consists of the following files:
 
-You can create template for the metadata files:
-- `metadata_table_columns.csv`: metadata for the postgres table fields
+- `<data_indicator>.R`: a script for downloading a dataset, processing it and preparing it for the postgres export 
+- `metadata_table_columns.csv`: metadata for the postgres table, describing the fields, besides `year` and `spatialunit_uid``
 - `metadata_tables.csv`: metadata for the postgres_table
+- `queries.sql`: Natural language questions and corresponding sql queries to answer these questions. This is the [Statbot Training Data]
+- `queries.log`: Log of the latest run of the sql queries either locally on the ds$postgres_export` or remote on the postgres database
+- `sample.csv`: sample of `ds$postgres_export` as csv file
 
-```
-statbotData::generate_metadata_templates(ds)
-```
+### Working on the pipelines
 
-This will create files `metadata_table_columns.template.csv` and 
-`metadata_tables.template.csv`: complete them and then rename them: 
+To work on a pipeline, open the file `<data_indicator>.R` and run the pipeline, adjust where needed. Once you are happy with `ds$postgres_export` you can open a script in the `scripts` directory to upload the dataset to postgres.
 
-```
-mv metadata_tables.template.csv metadata_tables.csv
-mv metadata_table_columns.template.csv `metadata_table_columns.csv
-```
+### More details on the file types in the pipelines
 
-#### Step 8: Generate templates for the metadata
+#### `queries.sql`
 
-Update the date in the metadata if you rerun the pipeline if needed:
+The `queries.sql` files contain the [Statbot Training Data](#statbot-training-data) for each dataset. See this section for a detailed description of these files.
 
-```
-statbotData::update_pipeline_last_run_date(ds)
-```
+#### `queries.log`
+
+These files are the output of a run of `statbotData::testrun_queries(ds)` on the queries in the `queries.sql` file of the dataset. A datatime indicates the datetime of the run. It contains also information whether the file was run on the remote postgres instance  `REMOTE` or on the local `ds$postgres` data frame.
+
+#### `metadata_tables.csv`
+
+These are metadata for a table in postgres. They are loaded into the postgres metadata table `metadata_tables`, see section on the [Statbot Postgres DB](#statbot-postgres-db) above. 
+
+#### `metadata_table_columns_.csv`
+
+These are metadata for the table columns in postgres. They are loaded into the postgres metadata table `metadata_table_columns`, see [Statbot Postgres DB](#statbot-postgres-db) for details. 
 
-Then upload the metadata to postgres: 
+## Scripts
 
+Originally the repo did not have a scripts directory and all the work like test running the queries and uploading metadata and data to postgres had been done directly in the pipelines. 
```

Comment:
```suggestion
```
**nitpick(non-blocking):** I think we could remove this to focus on the current status.